docs/user_guide/model-registration.rst

```diff
@@ -5,84 +5,60 @@
+proceeding with model registration.
 
-.. code-block:: yaml
+Option 1) Registration using an Excel template
+----------------------------------------------
 
-    model: model_a
-    native_regions:
-      - region_1: model_a|Region 1
-      - region_2: model_a|Region 2
-      - region_3: model_a|Region 3
-    common_regions:
-      - World:
-        - region_1
-        - region_2
-        - region_3
-      - Common region 1:
-        - region_1
-        - region_2
+Please use the `Excel template`_ and send it to the project managers by email.
 
-``model_a`` reports three regions natively, ``region_1``, ``region_2`` and ``region_3``.
-These three are to be renamed to the ``{model}|{region}`` pattern as is common place in
-many projects. For the ``common_regions``, there are ``World`` and ``Common region 1``.
+.. _`Excel template`: https://raw.githubusercontent.com/IAMconsortium/nomenclature/main/templates/model-registration-template.xlsx
 
-*For more details on the model-native and common regions refer to*
-:ref:`native-vs-common-region`.
+Option 2) Registration using a GitHub pull request
+--------------------------------------------------
 
-Region definitions
-^^^^^^^^^^^^^^^^^^
+The preferred approach for model registration is starting a GitHub pull request.
+Please contact the administrators if permissions for the project repository
+are required.
 
-In order to constitute a valid "model registration", regions ``model_a|Region 1``,
-``model_a|Region 2``, ``model_a|Region 3``, ``World`` and ``Common region 1`` **must**
-be part of the region definitions. 
+A model-registration pull request adds the following files (if required).
 
-``region_1``, ``region_2`` and ``region_3`` are **not required** since they refer to the
-input names of ``model_a``'s regions and will be renamed in the processing.
+Native-region definitions
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In most cases, the common regions (in the above example ``World`` and ``Common region
-1``) will already be defined in a file called ``definitions/region/regions.yaml``.
+If the model reports results at a model-specific regional resolution (e.g., other than
+national countries), add a file ``<your-model_vX.X>.yaml`` in the folder
+``definitions/region/model_native_regions/``.
 
-The model native regions, however, most likely need to be added. For this, a new yaml
-file should be created, usually in ``definitions/region/model_native_regions/``. The
-file does not need to have any special name but it is recommended to use the model name.
+This file should follow the region-naming conventions of :ref:`region`.
+The *hierarchy* property should be the model name inluding the version number.
 
-In order to complete the model registration for the example above, we would therefore
-add a file called ``model_a.yaml`` to ``definitions/region/model_native_regions/`` with
-the following content:
+Region-processing mapping
```

Comment:
I would call this a "model mapping" for consistency.

Reply:
Done